scripts/send-slack-notification-bot.py

```diff
@@ -61,15 +61,16 @@ def main():
         logging.error("❌ Error: Could not read version from cluster_version file.")
         sys.exit(1)
 
+    majorMinor = version.rsplit('.', 1)[0]
```

Comment:
just small nit, 
when working with versions it's better to create an object - cause it can break or have wired behaviour when using pre release or RC versions

example
```python
import semantic_version
majorMinor = semantic_version.Version(version).major

```

but I'm ok to approve this as is
/lgtm